src/spectrecoff/Output.fs

```diff
@@ -90,7 +91,8 @@ let appendNewline content =
     content + Environment.NewLine
 
 type OutputPayload =
-    | NewLine
+    | NextLine
+    | EmptyLine
```

Comment:
Maybe `BlankLine`  / `BL` is a better word, I thought later. Waht do you think, not so confusing with NewLine (which is like EndOfLine similar to EL)?

Reply:
Yeah, maybe it does make it a little clearer :) Done